src/Npgsql/NpgsqlTypes/NpgsqlDate.cs

```diff
@@ -69,7 +69,7 @@ internal NpgsqlDate(int days)
 
         public NpgsqlDate(DateTime dateTime) : this((int)(dateTime.Ticks / TimeSpan.TicksPerDay)) {}
 
-        public NpgsqlDate(NpgsqlDate copyFrom) : this(copyFrom._daysSinceEra) {}
+        public NpgsqlDate(in NpgsqlDate copyFrom) : this(copyFrom._daysSinceEra) {}
```

Comment:
Since `InternalType` has no specified underlying type, its size is 4 bytes (an `int` used to store a value). The total size of the struct is 8 bytes. No need in `in` here.

Reply:
removed use in modifier for NpgsqlDate